lib/graphql/schema.flow.js

```diff
@@ -564,14 +576,6 @@ export const DocumentTypeValues = Object.freeze({
 
 export type DocumentType = $Values<typeof DocumentTypeValues>;
 
-export type ElsterGroup = {|
```

Comment:
❓ Should this be gone ? Are we not using it anywhere?

Reply:
all of this is automatically generated from backend. Because backend has `elsterCode` in request/response, so it's generated like this.

Reply:
So, just to double check. Are we not using ElsterGroup anymore? 🤔

Reply:
no we don't, neither in `webapp` nor `app`